troubadix/standalone_plugins/version_updated.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+#  check_last_modification.py
+#
+# Copyright (C) 2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import subprocess
+import sys
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Iterable
+
+SCRIPT_VERSION_PATTERN = re.compile(
```

Comment:
Would it make sense to move these two pattern to helper/patterns.py or is there even some code included there for these two tags?

Moving could also help if there is some similarity with the pattern used in #312 which seems to be also about both tags.

Reply:
#314 is adding / has added such pattern to the `helper/patterns.py` which looks quite similar so might indeed worth to look if these could be used.

Reply:
I will look into it

Reply:
Done